tests/test_events.js

```diff
@@ -68,36 +66,36 @@ test('emits request and request body', async () => {
   expect(onReplied).to.have.been.calledOnce()
 })
 
-test('emits no match when no match and no mock', function(t) {
-  nock.emitter.once('no match', function() {
-    t.end()
+it('emits no match when no match and no mock', done => {
+  nock.emitter.once('no match', () => {
+    done()
   })
 
-  const req = http.get('http://example.test/abc')
-  req.once('error', ignore)
+  http.get('http://example.test/abc').once('error', ignore)
 })
 
-test('emits no match when no match and mocked', function(t) {
+it('emits no match when no match and mocked', done => {
   nock('http://example.test')
     .get('/')
     .reply(418)
 
-  const assertion = function(req) {
+  nock.emitter.on('no match', req => {
     expect(req.path).to.equal('/definitelymaybe')
     nock.emitter.removeAllListeners('no match')
```

Comment:
Could this be moved into an afterEach?

Reply:
I moved `removeAllListeners` to the global `afterEach`. I seemed like it belongs there.